conduit/conduit.cabal

```diff
@@ -59,7 +59,7 @@ test-suite test
                    , base
                    , hspec >= 1.3
                    , QuickCheck >= 2.7
-                   , transformers
+                   , transformers >= 0.4
```

Comment:
This constraint breaks Travis: https://travis-ci.org/snoyberg/conduit/jobs/246737024.

Can you either use conditional compilation to skip this code, or conditionally add a transformers-compat dependency for older GHCs?